src/dynamo.js

```diff
@@ -46,4 +48,28 @@ export class Dynamo {
     let params = { RequestItems: { OnlineUsers: userDeleteRequests } }
     await client.batchWriteItem(params).catch((e) => console.error('Error while deleting stale users', e))
   }
-}
\ No newline at end of file
+
+  async createGame(participants) {
+    const id = crypto.randomUUID()
+    const params = {
+      TableName: tableGames,
+      Item: marshall({
+        id: id,
+        participants: participants,
+        playerTurn: participants[Math.floor(Math.random())],
```

Comment:
`Math.floor(Math.random())` always returns 0,
I think you might want to do `Math.round(Math.random())`